### PR TITLE
chore(rxPageTitle): decompose monolithic source

### PR DESCRIPTION
**Pull request by CITguy**
JIRA: https://jira.rax.io/browse/FRMW-359

### LGTMS
- [x] Dev LGTM
- [x] Dev LGTM

### CODE REVIEW

File: src/rxPageTitle/scripts/rxPageTitle.js [hunk 1]
```diff
@@ -76,29 +61,4 @@ angular.module('encore.ui.rxPageTitle', [])
...
- * Given a string, it removes all HTML tags from the string, using the
- * browser's own parsing engine. Any content inside of tags will be kept.
- *
- * **NOTE:** You must only use this with **trusted text**. See this
- * {@link http://stackoverflow.com/a/5002618 StackOverflow} answer for more details.
- *
- * @param {String} htmlString The string to remove HTML from **trusted text**
- * @returns {String} Cleaned string
- */
-.filter('rxUnsafeRemoveHTML', function ($document) {
```
Comment (CITguy) [inline src/rxPageTitle/scripts/rxPageTitle.js]: moved to `src/rxPageTitle/scripts/rxUnsafeRemoveHTML.js`

File: src/rxPageTitle/scripts/rxPageTitle.spec.js [hunk 2]
```diff
@@ -91,52 +88,6 @@ describe('rxPageTitle', function () {
 
             pageTitle.setTitleUnsafeStripHTML(title);
             expect(pageTitle.getTitle()).to.equal(expectedTitle);
-            
         });
     });
 });
-
-describe('rxUnsafeRemoveHTML', function () {
```
Comment (CITguy) [inline src/rxPageTitle/scripts/rxPageTitle.spec.js]: moved to `src/rxPageTitle/scripts/rxUnsafeRemoveHTML.spec.js`

**Comment by 100stacks** [comment]
LGTM

**Comment by bry4n** [comment]
LGTM

**Pull request merged**